src/patternfly/components/Tooltip/tooltip.scss

```diff
@@ -20,6 +20,7 @@
   --pf-c-tooltip__arrow--m-left--Transform: translate(50%, -50%) rotate(45deg);
 
   position: relative;
+  display: inline-block;
```

Comment:
Same deal here as popover, I don't think we need `display` as it will be set by react. Here's what the component looks like currently with a small amount of text, shrinking to fit the content.

<img width="358" alt="screen shot 2019-02-26 at 12 30 48 pm" src="https://user-images.githubusercontent.com/35148959/53437120-6a37ef00-39c2-11e9-942a-8fc2832c702c.png">